src/Elasticsearch.Extensions.Logging/ElasticsearchLoggerOptions.cs

```diff
@@ -0,0 +1,71 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System;
+
+namespace Elasticsearch.Extensions.Logging
+{
+	public class ElasticsearchLoggerOptions
+	{
+		/// <summary>
+		/// Gets or sets the connection pool type. Default for multiple nodes is <c>Sniffing</c>; other supported values are
+		/// <c>Static</c>, <c>Sticky</c>, or force to <c>SingleNode</c>.
+		/// </summary>
+		public ConnectionPoolType ConnectionPoolType { get; set; }
+
+		/// <summary>
+		/// Gets or sets a flag indicating whether host details should be included in the message. Defaults to <c>true</c>.
+		/// </summary>
+		public bool IncludeHost { get; set; } = true;
+
+		/// <summary>
+		/// Gets or sets a flag indicating whether process details should be included in the message. Defaults to <c>true</c>.
+		/// </summary>
+		public bool IncludeProcess { get; set; } = true;
+
+		/// <summary>
+		/// Gets or sets a flag indicating whether scopes should be included in the message. Defaults to <c>true</c>.
+		/// </summary>
+		public bool IncludeScopes { get; set; } = true;
+
+		/// <summary>
+		/// Gets or sets a flag indicating whether user details should be included in the message. Defaults to <c>true</c>.
+		/// </summary>
+		public bool IncludeUser { get; set; } = true;
+
+		/// <summary>
+		/// Gets or sets the format string for the Elastic search index. The current <c>DateTimeOffset</c> is passed as parameter
+		/// 0.
+		/// </summary>
+		public string Index { get; set; } = "dotnet-{0:yyyy.MM.dd}";
```

Comment:
I don't know why other frameworks decided to use the same identifier as logstash. I know logstash is part of the ELK framework, but don't see any reason to masquerade as it. In fact it could be confusing if you also have logstash pushing events.  Maybe logstash is just that great that everyone wants to be them?

The value is configurable, so if there is some hard coded third party tool that is specifically looking for the logstash-* pattern, you can always change the index value.